net/gatling/Makefile

```
@@ -0,0 +1,40 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gatling
+PKG_VERSION:=0.16
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=https://www.fefe.de/gatling/gatling-0.16.tar.xz
```

```suggestion
PKG_SOURCE_URL:=https://www.fefe.de/gatling/
```

Shouldn't this work too?

---

Compile-tested it. Works fine.